work-in-progress/typical_meteorological_year_methodology.ipynb

```diff
@@ -1201,7 +1181,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.12.8"
+   "version": "3.12.10"
```

Comment:
would love to move to an even newer version if we can, but not a big deal.

Reply:
Weird, my local version shows 3.12.10 when I open the notebook in an editor but that seems to not be getting tracked in git